Google.Api.Generator.Tests/ProtoTests/Basic.v1/Testing.Basic.V1.StandaloneSnippets/Testing.Basic.V1.StandaloneSnippets.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd expect the project files to be renames too - perhaps you didn't delete them from Testing.Basic (or didn't add the delete to Git?)

Reply:
Didn't add the delete to Git. Done now.